articles/understanding-systemd/index.md

```diff
@@ -0,0 +1,248 @@
+## Introduction
+From the official website, systemd (intentionally spelt with a small 's') is described as a suite of tools that forms the basic building block for a Linux system.
+systemd is primarily responsible for handling the boot process for Linux systems. As an init implementation, it has a PID of 1 just like other init systems e.g System V, Upstart.
+
+### What's init?
+Before we delve further, lets talk more on what exactly init is. 
+An init program is usually the first to start and the last to stop. It initializes and terminates programs and services crucial to the operation of Linux systems. It's actually a daemon process that runs during the entire life time of all other programs and services in the system. It's acts as the parent process such that when child processes are orphaned or become zombies, the init process takes the responsibility of ensuring that they're properly terminated. 
+Now, what happens when the init process doesn't start first (the kernel doesn't boot the init process)? A kernel panic occurs because there isn't an init process to startup things and make way for essential services.
```

Comment:
A few modifications would make this paragraph much better. Explain 'daemon' briefly, or add a link to any resource about daemon. Better still, add requirements (knowledge required to learn systemd) for any user about to read this article.